test/modern_bpf/test_suites/syscall_enter_suite/pipe2_e.cpp

```diff
@@ -0,0 +1,44 @@
+#include "../../event_class/event_class.h"
+
+#ifdef __NR_pipe2
+TEST(SyscallEnter, pipe2E)
```

Comment:
This commit lacks the bpf program? Is it separately or in a different commit?

Reply:
the rationale here is that we use the same BPF programs for more than one syscalls if they are similar like in this case `pipe2` and `pipe` ( i left a comment on that [here](https://github.com/falcosecurity/libs/pull/518/files/1464810680abdf8649114a6e86df47e844496d77..ad507dfcc84a4e1e232c07200c626b53caa67cda#diff-7f600dfb3d1374b96ee59872c9017dc87875ef9d842a2f0aeb61da8ecb772e09R6-R9)). BTW, I don't like this approach to much since in this way we cannot catch the flags of `pipe2` for example, unfortunately, this is a known issue :( @FedeDP 
I will add it to the tracking issue :)

Reply:
Thanks for the explanation! And thanks for the comment. Agree on the flags arg, so :+1: for the tracking issue.